android/src/main/java/com/oney/WebRTCModule/GetUserMediaImpl.java

```diff
@@ -141,16 +118,30 @@ private void checkMandatoryConstraints(MediaConstraints peerConstraints) {
         peerConstraints.mandatory.addAll(valid);
     }
 
+    private CameraEnumerator getCameraEnumerator() {
+        if (cameraEnumerator == null) {
+            if (Camera2Enumerator.isSupported(reactContext)) {
```

Comment:
Note: the astute reader might notice we no longer try-except this. It used to throw, but not anymore, it was fixed upstream.